tgui/packages/tgui/interfaces/JobSelection.tsx

```diff
@@ -0,0 +1,182 @@
+import { useBackend } from '../backend';
+import { Button, Stack, Box, Icon } from '../components';
+import { Window } from '../layouts';
+import { JOB2ICON } from './Orbit/constants';
```

Comment:
make it a shared module

Reply:
No idea what that funny word means.

Reply:
you should not be including files from a different interface. it should be in some shared location, like tgui/data/jobs or similar. prefs menu could easily use this for instance

Reply:
Ah, noted!